web/problems/templates/python/check.py

```diff
@@ -1,5 +1,7 @@
 import io
+from _thread import interrupt_main
```

Comment:
To je malo riskantno, ker gre za zasebno funkcijo. Kaj pa, če bi si spodaj definirali svojo funkcijo, ki sproži `TimeoutError`?

Reply:
Težava je v tem, da bi `TimeoutError` zmotil le thread, kjer je bil sprožen; želimo pa prekiniti glavni thread.
Problem je rešen z uporabo `signal.raise_signal`, ki naredi isto kot `interrupt_main`.